### PR TITLE
Add cucushift-installer-rehearse-vsphere-ipi-sdn profile from QE to c…

### PR DESCRIPTION
**Pull request by miyadav**
…luster-bot workflow-launch.

@sunzhaohua2, @huali9 , @jhou1  could you PTAL when time permits , wanted to debug test failed on the profile .
Adding @liangxia, @jinyunma in case similar profile already present and not needed to add this one.

### CODE REVIEW

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 1]
```diff
@@ -150,6 +150,8 @@ workflows:
     platform: vsphere
   cucushift-installer-rehearse-vsphere-upi-proxy-https:
     platform: vsphere
+  cucushift-installer-rehearse-vsphere-ipi-sdn:
```
Comment (liangxia) [inline core-services/ci-chat-bot/workflows-config.yaml:153]: sort them

**Comment by sunzhaohua2** [comment]
I am not sure, I think the default is SDN, so maybe you added  is same as below?
```
  cucushift-installer-rehearse-vsphere-ipi:
    platform: vsphere
```

File: core-services/ci-chat-bot/workflows-config.yaml (hunk 1 above)
Comment (miyadav) [reply core-services/ci-chat-bot/workflows-config.yaml]: done

**Comment by miyadav** [comment]
> I am not sure, I think the default is SDN, so maybe you added is same as below?
> 
> ```
>   cucushift-installer-rehearse-vsphere-ipi:
>     platform: vsphere
> ```
I see this workflow - `cucushift-installer-rehearse-vsphere-ipi-sdn-workflow.yaml` so may not be default , may be @jinyunma can confirm ?

**Comment by liangxia** [comment]
> I am not sure, I think the default is SDN, so maybe you added is same as below?
> 
> ```
>   cucushift-installer-rehearse-vsphere-ipi:
>     platform: vsphere
> ```

SDN is default in 4.11 and lower, and OVN is the default in 4.12+

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**